packages/web-components/src/components/masthead/megamenu-top-nav-menu.ts

```diff
@@ -76,6 +82,7 @@ class DDSMegaMenuTopNavMenu extends DDSTopNavMenu {
 
   updated(changedProperties) {
     super.updated(changedProperties);
+    this._menuNode.removeAttribute('role');
```

Comment:
Would you do this in `firstUpdated()` given it doesn't have to do with a property update?

Reply:
ah yes! will do